AltServer/AppDelegate.swift

```diff
@@ -68,28 +62,6 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 
 private extension AppDelegate
 {
-    @objc func presentMenu()
-    {
-        guard let button = self.statusItem?.button, let superview = button.superview, let window = button.window else { return }
-        
-        self.connectedDevices = ALTDeviceManager.shared.connectedDevices
-        
-        self.launchAtLoginMenuItem.state = LaunchAtLogin.isEnabled ? .on : .off
-        self.launchAtLoginMenuItem.action = #selector(AppDelegate.toggleLaunchAtLogin(_:))
-        
-        let x = button.frame.origin.x
-        let y = button.frame.origin.y - 5
```

Comment:
I think you could have simply removed the `-5` here to make it appear attached without making the rest of these changes.

Reply:
No, that doesn't do the same thing. I'll upload screenshots in a bit.

Reply:
master:

<img width="218" alt="Screen Shot 2020-01-07 at 8 59 40 PM" src="https://user-images.githubusercontent.com/5678977/71952166-9ac7e800-3192-11ea-9d36-226046f829ca.png">

master with the -5 removed:

<img width="221" alt="Screen Shot 2020-01-07 at 9 13 27 PM" src="https://user-images.githubusercontent.com/5678977/71952168-9ac7e800-3192-11ea-861b-24ac70c56c70.png">

This PR:

<img width="239" alt="Screen Shot 2020-01-07 at 9 16 27 PM" src="https://user-images.githubusercontent.com/5678977/71952290-00b46f80-3193-11ea-8276-72a6c9983348.png">

First two definitely look wrong to me.